ci/cleanup-resources.sh

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Same as for end-to-end-test.sh, this line causes the script to fail:
```
 ./cleanup-resources.sh 
./cleanup-resources.sh: line 1: ---: command not found
```
Probably removing or adding it as comment should help here.